scipy/stats/_multivariate.py

```diff
@@ -520,19 +582,19 @@ def __init__(self, mean=None, cov=1, allow_singular=False):
         """
         self.dim, self.mean, self.cov = _process_parameters(None, mean, cov)
         self.cov_info = _PSD(self.cov, allow_singular=allow_singular)
-        self._mnorm = multivariate_normal_gen()
+        self.dist = multivariate_normal_gen(seed)
 
```

Comment:
This technically makes `self.dist` a public attribute since it does not have a leading underscore. Could you revert this change here and elsewhere? I'd think we do not need to emulate a legacy decision made for univariate frozen distributions.